### ISSUE TITLE
Optional route parameters

### ISSUE THREAD

**Issue by rajeshaz09** [domaindrivendev/Swashbuckle.AspNetCore#2371]
### Problem
Optional route parameters specified for a controller is not reflecting in the UI. Lets say if I add **Route** attribute to the controller
`
[Route("api/[controller]/[action]/{id?}")]
`
When I open swagger UI page, it is not allowing me to test API without specifying the **Id**.

![image](https://user-images.githubusercontent.com/635263/157641915-fed58df2-afb6-4e43-b75f-86b9dfd4bc88.png)

**Comment by domaindrivendev** [comment]
Dup of #406. TL/DR: Swashbuckle does not currently support "optional" path parameters because the OpenAPI specification does not support optional path parameters:

> If the parameter location is "path", this property is REQUIRED and its value MUST be true

**Comment by netotz** [comment]
TL;DR: [use 2 controller methods](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/406#issuecomment-1431393820) or [remove the parameter from Route string](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/406#issuecomment-1581669927)
